src/components/calcite-radio-button-group/usage/basic.md

```diff
@@ -0,0 +1,8 @@
+```html
+<calcite-radio-button-group name="group-basic">
+  <calcite-radio-button value="maps" checked>Maps</calcite-radio-button>
+  <calcite-radio-button value="layers">Layers</calcite-radio-button>
+  <calcite-radio-button value="data">Data</calcite-radio-button>
+  <calcite-radio-button value="geography">Geography</calcite-radio-button>
```

Comment:
`calcite-radio-button` no longer supports slotting in label content, so you should add wrapping `calcite-label`s with `layout="inline"` on them here.

Reply:
done!